docs/pages/toast.js

```diff
@@ -1,7 +1,5 @@
 // @flow strict
```

Comment:
Holding off on converting the docs page to the new style for now, keeping this PR a bit more focused